django/db/models/base.py

```diff
@@ -230,8 +230,13 @@ def __new__(cls, name, bases, attrs):
                     field = parent_links[base_key]
                 elif not is_proxy:
                     attr_name = '%s_ptr' % base._meta.model_name
-                    field = OneToOneField(base, name=attr_name,
-                            auto_created=True, parent_link=True)
+                    field = OneToOneField(

```

Comment:
It seems some Windows line endings crept in here? I discovered this by accident and didn't audit other files.